src/test/java/no/nav/fo/veilarbregistrering/oppgave/OppgaveServiceTest.java

```diff
@@ -71,27 +73,28 @@ public void opprettOppgave_ang_dod_utvandret_skal_gi_beskrivelse_om_rutine() {
                 "Brukeren får ikke registrert seg som arbeidssøker fordi bruker står som utvandret i Arena, " +
                 "og har selv opprettet denne oppgaven. " +
                 "Ring bruker og følg vanlig rutine for slike tilfeller.",
-                LocalDate.of(2020, 4, 15));
+                LocalDate.of(2020, 4, 15),
+                LocalDate.of(2020, 4, 10));
     }
 
     @Test
     public void skal_lagre_oppgave_ved_vellykket_opprettelse_av_oppgave() {
         when(oppgaveRouter.hentEnhetsnummerFor(BRUKER, OPPHOLDSTILLATELSE)).thenReturn(Optional.empty());
-        when(oppgaveGateway.opprettOppgave(any(), any(), any(), any())).thenReturn(new DummyOppgaveResponse());
+        when(oppgaveGateway.opprettOppgave(any(), any(), any(), any(), any())).thenReturn(new DummyOppgaveResponse());
         oppgaveService.opprettOppgave(BRUKER, OPPHOLDSTILLATELSE);
 
         verify(oppgaveRepository, times(1))
                 .opprettOppgave(BRUKER.getAktorId(), OPPHOLDSTILLATELSE, 234L);
     }
 
-    @Test(expected = Feil.class)
+    @Test
     public void skal_kaste_exception_dersom_det_finnes_nyere_oppgave_fra_for() {
         OppgaveImpl oppgaveSomBleOpprettetDagenFor = new OppgaveImpl(23, BRUKER.getAktorId(), OPPHOLDSTILLATELSE, 23, LocalDateTime.of(2020, 4, 9, 22, 0));
         List<OppgaveImpl> oppgaver = Collections.singletonList(oppgaveSomBleOpprettetDagenFor);
 
         when(oppgaveRepository.hentOppgaverFor(any())).thenReturn(oppgaver);
 
-        oppgaveService.opprettOppgave(BRUKER, OPPHOLDSTILLATELSE);
+        assertThrows(Feil.class, () -> oppgaveService.opprettOppgave(BRUKER, OPPHOLDSTILLATELSE));
```

Comment:
JUnit 5 har fått litt bedre støtte for å håndtere exception. Her kan vi også lage en variabel hvor vi henter ut message om vi vil.